Backend/models/User.js

```diff
@@ -0,0 +1,26 @@
+const mongoose = require('mongoose');
+const { Schema } = mongoose;
+
+const UserSchema = new Schema({
+     name: {
+        type: String,
+        required: true,
+     },
+
+     email: {
+        type: String,
+        required: true,
+        unique: true,
+     },
+
+     password: {
+        type: String,
+        required: true,
+     },
+     date: {
```

Comment:
date is fine, but its should be more meaningful in terms as in created_at or updated_at

Reply:
got it.
Thankyou !